mopidy_pummeluff/actions/tracklist.py

```diff
@@ -34,7 +51,16 @@ def execute(cls, core, uri):  # pylint: disable=arguments-differ
         if uri in playlists:
             uris = [item.uri for item in core.playlists.get_items(uri).get()]
         else:
-            uris = [uri]
+            # if uri points to a Directory, we have to call browse to get all
+            # the Tracks. Right now it is not possible to get the type of an
+            # URI. So we just try to browse uri and if it is not a directory
+            # this will return an empty list of uris.
+            uris = cls._browse_uri(core, uri)
```

Comment:
@ueb Are you sure this is working? Because the method doesn't seem to return anything :/

Reply:
Sorry, I did test the wrong branch.

It is working now. Tested it with:
* a single file
* a directory without subdirectories
* a driectory with subdirectories

Reply:
Can you please ensure the tests are running successful:

```
pylint mopidy_pummeluff
************* Module mopidy_pummeluff.actions.tracklist
mopidy_pummeluff/actions/tracklist.py:26:0: C0303: Trailing whitespace (trailing-whitespace)
mopidy_pummeluff/actions/tracklist.py:28:31: C0303: Trailing whitespace (trailing-whitespace)
mopidy_pummeluff/actions/tracklist.py:62:0: C0303: Trailing whitespace (trailing-whitespace)
mopidy_pummeluff/actions/tracklist.py:63:8: W1202: Use % formatting in logging functions and pass the % parameters as arguments (logging-format-interpolation)
************* Module mopidy_pummeluff.actions.__init__
mopidy_pummeluff/actions/__init__.py:1:0: R0801: Similar lines in 2 files
==mopidy_pummeluff.actions.__init__:4
==mopidy_pummeluff.actions.playback:4
__all__ = (
    'PlayPause',
    'Stop',
    'PreviousTrack',
    'NextTrack', (duplicate-code)
```